index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <a href="/" class="img_logo">
+        <img src="images/Img_logo.png" alt="logo image" />
+      </a>
+      <button onclick="location.href='/signin'" class="login_button">
+        로그인
+      </button>
+    </header>
+    <main>
+      <section class="banner">
+        <div class="banner_container">
+          <div class="banner_content">
+            <h1 class="banner_title">
+              일상의 모든 물건을 <br />
+              거래해 보세요
+            </h1>
+            <button onclick="location.href='/items'" class="look_button">
```

Comment:
html에서도 문자열로 이런 처리가 가능하다는 것을 알고 시도해보신 부분은 좋습니다 👍 

의미적으로는 다른 경로로 이동시키는 요소니까 anchor 태그가 더 적절하고
표현면에서도 html에서 onclick handler를 문자열로 작성하기보다, 진짜 의도인 '이동시킬 경로'만 나타내는게 의도를 잘 드러낼 거같아요.

다만 a 태그의 기본 스타일도 있어서 약간 처리가 필요할 수 있겠죠. 시각적인 것, 스타일보다는 시맨틱하게 작성하는 것을 우선으로하고, a태그의 기본 스타일은 직접 없애두고 css를 작성하거나 reset.css 를 활용해볼 수 있겠습니다

```suggestion
            <a  href="/items'" class="look_button">
```